changelog.d/309.feature

```diff
@@ -0,0 +1 @@
+Improve GitLab push hook formatting
```

Comment:
Can you expand on this, something about only showing one commit if this goes over the limit.